roadmap/implementers-guide/src/node/availability/bitfield-signing.md

```diff
@@ -4,6 +4,10 @@ Validators vote on the availability of a backed candidate by issuing signed bitf
 
 ## Protocol
 
+Input:
+
+There is no dedicated input mechanism for bitfield signing. Instead, Bitfield Signing produces a bitfield representing the current state of availability on `StartWork`.
```

Comment:
Describing things that are _not_ there is really helpful when reading the guide :+1: